KHJ/4Day/로또의 최고순위와 최저순위.js

```diff
@@ -0,0 +1,14 @@
+function solution(lottos, win_nums) {
+    let zeroCount = 0;
+    let matchCount = 0; 
+    for (let i = 0; i < lottos.length; i++) {
+        if (lottos[i] === 0) {
+            zeroCount++;
+        } else if (win_nums.includes(lottos[i])) {
+            matchCount++;
+        }
+    }
+    const maxRank = Math.max(1, 7 - (matchCount + zeroCount)); 
+    const minRank = Math.max(1, 7 - matchCount); 
+    return [maxRank, minRank].map(rank => rank === 7 ? 6 : rank);
+}
```

Comment:
```suggestion
function solution(lottos, win_nums) {
    let zeroCount = 0;
    let matchCount = 0; 
    
    for (let i = 0; i < lottos.length; i++) {
        if (lottos[i] === 0) {
            zeroCount++;
        } else if (win_nums.includes(lottos[i])) {
            matchCount++;
        }
    }
    
    const maxRank = Math.max(1, 7 - (matchCount + zeroCount)); 
    const minRank = Math.max(1, 7 - matchCount); 
    
    return [maxRank, minRank].map(rank => rank === 7 ? 6 : rank);
}
```
@eric5025 
변수 선언부와 for/while/if문 등은 코드 맥락에 따라 줄바꿈으로 나눠주는게 가독성을 늘리는 팁입니다.
현재 코드를 보시면 따닥따닥 붙어있으니 지금이야 코드가 짧아서 큰 문제는 없지만 코드 길이가 늘어난다면 보기가 무척 어려워질거에요 ㅎㅎ

Reply:
네 ㅠㅠ 줄바꿈 신경써야겠어요